andela/vale-ah-backend
# #164139768 Send verification email 164139768

**What does this PR do?**
Sends verification emails to users after successful registration

**Description of Task to be completed?**
- Send mail to signed up user
-  add endpoint to verify User accounts

**How should this be manually tested?**
- Checkout `ft-send-verification-email-164139768`
- Run `npm i`
- Create a new user account by sending a valid post request to `/api/users`
  A successfully created user should receive a verification email.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
#164139768

**Screenshots (if appropriate)**
